lib/src/signed_video_h26x_sign.c

```diff
@@ -676,6 +690,37 @@ signed_video_set_end_of_stream(signed_video_t *self)
   return svi_rc_to_signed_video_rc(status);
 }
 
+SignedVideoReturnCode
+signed_video_generate_golden_sei(signed_video_t *self)
+{
+  if (!self) return SV_INVALID_PARAMETER;
+
+  uint8_t *payload = NULL;
+  uint8_t *payload_signature_ptr = NULL;
+  svi_rc status = SVI_UNKNOWN;
```

Comment:
Put this just before `SVI_TRY()` since it belongs to the try-catch scope. Also add an empty line before it to mark a clear separation.